doc/classes/BaseMaterial3D.xml

```diff
@@ -610,7 +610,7 @@
 			The color of the object is multiplied by the background.
 		</constant>
 		<constant name="BLEND_MODE_PREMULT_ALPHA" value="4" enum="BlendMode">
-			The color of the object is added to the background and the alpha channel is used to mask out the background. This is effectively a hybrid of the blend mix and add modes, useful for FX like fire where you want the flame to add but the smoke to mix. By default, this works with unshaded materials using premultiplied textures. For shaded materials, use the PREMUL_ALPHA_FACTOR built-in so that lighting can be modulated as well.
+			The color of the object is added to the background and the alpha channel is used to mask out the background. This is effectively a hybrid of the blend mix and add modes, useful for FX like fire where you want the flame to add but the smoke to mix. By default, this works with unshaded materials using premultiplied textures. For shaded materials, use the PREMULT_ALPHA_FACTOR built-in so that lighting can be modulated as well.
```

Comment:
```suggestion
			The color of the object is added to the background and the alpha channel is used to mask out the background. This is effectively a hybrid of the blend mix and add modes, useful for effects like fire where you want the flame to add but the smoke to mix. By default, this works with unshaded materials using premultiplied textures. For shaded materials, use the [code]PREMULT_ALPHA_FACTOR[/code] built-in so that lighting can be modulated as well.
```